.claude/commands/issue.md

```
@@ -0,0 +1,168 @@
+#### 実装の場合
+- ブランチ名: `feature/issue-{Issue番号}-{簡潔な説明}`
+- 例: `feature/issue-123-add-player-jump`
+
+#### 修正の場合
+- ブランチ名: `fix/issue-{Issue番号}-{簡潔な説明}`
+- 例: `fix/issue-123-add-player-jump`
+
+```bash
+git checkout -b {ブランチ名}
+```
+
```

markdownlint-cli2 (0.20.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)